deployment/marketplace/digitialocean/one-click-droplet/Makefile

```diff
@@ -0,0 +1,8 @@
+RELEASE_NAME := vm-oneclick-droplet
+VM_VERSION ?= $(shell git describe --abbrev=0 --tags)
+
+.PHONY: $(MAKECMDGOALS)
+
+release-victoria-metrics-digitalocean-oneclick-droplet:
+	sed -i -e "s/VM_VERSION/${VM_VERSION}/g" ./files/etc/update-motd.d/99-one-click
```

Comment:
looks like it will work only one time...
should we `cp` that file and modify it then? or having smthng like a template and compiled result?

Reply:
fixed